pyroSAR/drivers.py

```diff
@@ -1746,6 +1749,38 @@ def __init__(self, dbfile, custom_fields=None, postgres=False, user='postgres',
         sys.stdout.write('\rchecking for missing scenes..done\n')
         sys.stdout.flush()
     
+    def add_tables(self, tables, verbose=False):
+        """
+        Add tables to the database per :class:`sqlalchemy.schema.Table`
+        Tables provided here will be added to the database. Notice that columns using Geometry must have setting
+        management=True for SQLite, for example: bbox = Column(Geometry('POLYGON', management=True, srid=4326))
+        
+        Parameters
+        ----------
+        tables: :class:`sqlalchemy.schema.Table` or :obj:`list` of :class:`sqlalchemy.schema.Table`
```

Comment:
please add intersphinx mapping to [docs.conf.py](https://github.com/MarkusZehner/pyroSAR/blob/sqlalchemy_postgres/docs/conf.py) and check that sphinx documentation is properly rendered with  
`python setup.py build_sphinx`.